testbed/tests/trace_test.go

```diff
@@ -65,6 +65,15 @@ func TestTrace10kSPS(t *testing.T) {
 				ExpectedMaxRAM: 84,
 			},
 		},
+		{
+			"Zipkin",
+			testbed.NewZipkinDataSender(testbed.GetAvailablePort(t)),
+			testbed.NewZipkinDataReceiver(testbed.GetAvailablePort(t)),
+			testbed.ResourceSpec{
+				ExpectedMaxCPU: 42,
```

Comment:
The values for `ExpectedMaxCPU` and `ExpectedMaxRAM` are the things which I'm not sure.

Reply:
The values need to be determined after running on TravisCI. We use (max-observed * 1.3) formula for limits.